websites/H/HarmonyRadio/dist/metadata.json

```
@@ -0,0 +1,21 @@
+{
+    "author": {
+      "name": "Milan",
+      "id": "700817449414557736"
+    },
+    "service": "Harmony",
+    "description": {
+      "en": "Harmony | You bring the Harmony to music."
+    },
+    "url": [
+      "weareharmony.net"
+    ],
```

Only one URL so this will do: 
```suggestion
    "url":
      "weareharmony.net"
```